packages/bicep-deploy-common/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "bicep-deploy-common",
+  "description": "Common Code for bicep-deploy (GitHub Action/ADO Task)",
+  "version": "0.0.1",
```

Comment:
Do you have a plan for how this version # will get incremented for a release?

Reply:
Got it. I think it would be worth having a discussion and documenting how this should work.

The release process for the action is here: https://github.com/Azure/bicep-deploy/blob/main/scripts/release.sh, however we may want to version the package and action separately, as there are downsides to being too eager with bumping versions for the action. See #141 for example - I was not aware of this when releasing v2!